### DISCUSSION TITLE
Versions of some nuget

### DISCUSSION THREAD

**Discussion by EricBui619** [OrchardCMS/OrchardCore#9313] [Q&A]
Hi all, I wanna implement custom logic for my GraphQl query based on source code of OC. I borrow some logic from the one but I got problem is that the difference between version of nugets. The lastest version of YesSql is 3.0.2 but OC rc2-13450 is using yessql 2.0.0 beta-1481. So some methods such as SqlDialect, propertyProviders paremeter in PredicateQuery...are missing. How can I synchronize them to be able to clone features from OC to my project?

![image](https://user-images.githubusercontent.com/30250345/116499209-d16cc900-a8d5-11eb-814e-9178a97bfa12.png)

![image](https://user-images.githubusercontent.com/30250345/116499558-a2a32280-a8d6-11eb-9418-e482495af519.png)

**Comment by hishamco** [comment]
It might be related to #9059

**Comment by deanmarcussen** [comment]
The suggestion would be to either upgrade to the dev feed of OrchardCore, or take the example code from master which is the same version as RC2-13450

https://docs.orchardcore.net/en/dev/docs/getting-started/preview-package-source/

**Reply by EricBui619** [reply]
But I wonder how stable the dev feed is? Because I will apply it for our real project
